examples/ITK_Example20_PixelTypes.ipynb

```diff
@@ -0,0 +1,325 @@
+    "transform_parameters_short.SetParameter(0, 'FinalBSplineInterpolationOrder', '1')\n",
+    "\n",
+    "# Set transform to zero so that the input image matches the output image\n",
+    "transform_parameters_float.SetParameter(0, 'TransformParameters', 6 * ['0']) # 6 transformation parameters to zero\n",
+    "transform_parameters_short.SetParameter(0, 'TransformParameters', 6 * ['0'])\n",
+    "\n",
+    "print(transform_parameters_float)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "7d610c09",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "For float internal pixel type - Diff min: 0.0 | max: 0.0\n",
+      "For short internal pixel type - Diff min: -0.9999923706054688 | max: -6.811788807681296e-06\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Create the input image as float\n",
+    "image = create_random_image(np.float32)\n",
+    "\n",
+    "# Transform the image using 'float' internal pixel type\n",
+    "transformed_image_float = itk.transformix_filter(image,\n",
+    "                                                 transform_parameter_object=transform_parameters_float,\n",
+    "                                                )\n",
+    "\n",
+    "# Transform the image using 'short' internal pixel type\n",
+    "transform_parameters_short.SetParameter(0, 'FixedInternalImagePixelType', 'short')\n",
+    "transform_parameters_short.SetParameter(0, 'MovingInternalImagePixelType', 'short')\n",
+    "transformed_image_short = itk.transformix_filter(image,\n",
+    "                                                 transform_parameter_object=transform_parameters_short,\n",
+    "                                                )\n",
+    "\n",
+    "# Compute the differences between the result and the input image\n",
+    "diff_float = transformed_image_float[:] - image[:]\n",
+    "diff_short = transformed_image_short[:] - image[:]\n",
+    "\n",
+    "print(f\"For float internal pixel type - Diff min: {diff_float.min()} | max: {diff_float.max()}\")\n",
+    "print(f\"For short internal pixel type - Diff min: {diff_short.min()} | max: {diff_short.max()}\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "d40cf32a",
+   "metadata": {},
+   "source": [
+    "We observe that difference between input and output images is zero. On the other hand, for the case of `short` internal type we see that there is a difference that corresponds to the rounding error (truncation of the floating part). "
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "f675ed9e",
+   "metadata": {},
+   "source": [
+    "Note: Since specifying the internal pixel type to short is meant as a wayt reduce the memory requirements for the registration, elastix supports it by default only for `>=3D` images. There is the option to enable support for `short` with `2D` images when building elastix on your own."
```

Comment:
Little typo: "a wayt reduce" --> "a way to reduce"

I understand if you don't want too much details, but maybe still mention that it's a _CMake_ option:

> There is a CMake option to enable support for `short` with `2D` images when building elastix on your own.